mongoose.h

```diff
@@ -1107,6 +1107,7 @@ char *mg_random_str(char *buf, size_t len);
 uint32_t mg_crc32(uint32_t crc, const char *buf, size_t len);
 uint64_t mg_millis(void);  // Return milliseconds since boot
 bool mg_path_is_sane(const struct mg_str path);
+void mg_delayms(unsigned int ms);
```

Comment:
I suggest NOT to add this function to mongoose.h

instead, how about adding it as `static inline` only to the hal.h which require it ?